prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Patricia Manalang';
+    console.log(fullName);
+
+    var numberOfPets = '0';
+    console.log(numberOfPets);
+
+    var likesVegetables = 'true';
+    console.log(likesVegetables);
+
+    var nothing = 'null';
```

Comment:
Hi Patricia, be sure to leave off the quotes from numbers, booleans, and null. Otherwise the JavaScript engine will interpret them as strings.